src/features/unified-share-modal/SharedLinkSection.js

```diff
@@ -384,7 +385,21 @@ class SharedLinkSection extends React.Component<Props, State> {
                         <span className="security-indicator-icon-globe">
                             <IconGlobe height={12} width={12} />
                         </span>
-                        {permissionLevel === CAN_EDIT ? (
+                        // TODO: temporary change to support Canvas not being truly public
```

Comment:
how are you able to add this comment here? wouldn't it have a linting error?

Reply:
Not sure 😮 but didn't get any errors; updated